src/web/user_agent.rs

```diff
@@ -48,6 +49,15 @@ impl fmt::Display for OsFamily {
 pub struct DeviceInfo {
     pub form_factor: FormFactor,
     pub os_family: OsFamily,
+    // We only care about major versions.
+    pub ff_version: u32,
+}
+
+impl DeviceInfo {
+    pub fn legacy_only(&self) -> bool {
+        self.os_family == OsFamily::IOs && self.ff_version < 36
```

Comment:
We're serving Desktop only, was there a reason for checking less than < 36 on iOS? (I guess it doesn't matter much either)

Reply:
The fact that both versions differ was only noted when I updated the UA strings for testing. I'm fine saying desktop only for this check but I'd like to prevent potential future issues. In this case "You Hope You Ain't Gonna Need It".